urdfenvs/urdfCommon/urdfEnv.py

```diff
@@ -92,70 +169,132 @@ def step(self, action):
+        """Compose the observation."""
         observation = self.robot.get_observation()
         if not self.observation_space.contains(observation):
-            err = WrongObservationError("The observation does not fit the defined observation space", observation, self.observation_space)
+            err = WrongObservationError(
+                "The observation does not fit the defined observation space",
+                observation,
+                self.observation_space,
+            )
             warnings.warn(str(err))
         return observation
 
-    def addObstacle(self, obst):
+    def addObstacle(self, obst) -> None:
+        """Adds obstacle to the simulation environment.
+
+        Parameters
+        ----------
+
+        obst: Obstacle from MotionPlanningEnv
+        """
         # add obstacle to environment
         self._obsts.append(obst)
         obst.add2Bullet(p)
 
         # refresh observation space of robots sensors
-        sensors = self.robot.getSensors()
+        sensors = self.robot.sensors()
         curDict = dict(self.observation_space.spaces)
         for sensor in sensors:
             curDict[sensor.name()] = sensor.getObservationSpace()
         self.observation_space = gym.spaces.Dict(curDict)
 
-        if  self._t != 0.0:
-            warnings.warn("Adding an object while the simulation already started")
+        if self._t != 0.0:
+            warnings.warn(
+                "Adding an object while the simulation already started")
 
-    def getObstacles(self):
+    def getObstacles(self) -> list:
         return self._obsts
 
-    def addGoal(self, goal):
+    def addGoal(self, goal) -> None:
+        """Adds goal to the simulation environment.
+
+        Parameters
+        ----------
+
+        goal: Goal from MotionPlanningGoal
+        """
         self._goals.append(goal)
         goal.add2Bullet(p)
 
-    def setWalls(self, limits=[[-2, -2], [2, 2]]):
-        colwallId = p.createCollisionShape(p.GEOM_BOX, halfExtents=[0.05, 10.0, 0.5])
-        p.createMultiBody(0, colwallId, 10, [limits[0][0], 0, 0.0], p.getQuaternionFromEuler([0, 0, 0]))
-        p.createMultiBody(0, colwallId, 10, [limits[1][0], 0, 0.0], p.getQuaternionFromEuler([0, 0, 0]))
-        p.createMultiBody(0, colwallId, 10, [0, limits[0][1], 0.0], p.getQuaternionFromEuler([0, 0, np.pi/2]))
-        p.createMultiBody(0, colwallId, 10, [0, limits[1][1], 0.0], p.getQuaternionFromEuler([0, 0, np.pi/2]))
+    def setWalls(self, limits: list = [[-2, -2], [2, 2]]) -> None:
+        """Adds walls to the simulation environment.
+        # TODO: To be removed in the future and incorporated
+        into addObstacle <10-03-22, maxspahn> #
+
+        Parameters
+        ----------
 
-    def addSensor(self, sensor):
+        limits: Positions of walls, [[x_low, y_low], [x_high, y_high]]
+        """
+        colwallId = p.createCollisionShape(
+            p.GEOM_BOX, halfExtents=[0.05, 10.0, 0.5])
+        p.createMultiBody(
```

Comment:
here a reason to use this fold out format here instead of the previous oneline function call? makes the code quite long.

Reply:
The line would be 85 characters long without the folding.
That is too long according to PEP8 guidelines.